yt_dlp/extractor/rumble.py

```diff
@@ -75,3 +76,35 @@ def _real_extract(self, url):
             'channel_url': author.get('url'),
             'duration': int_or_none(video.get('duration')),
         }
+
+
+class RumbleChannelIE(InfoExtractor):
+    _VALID_URL = r'(?P<url>https?://(?:www\.)?rumble\.com/(?:c|user)/(?P<id>[^&?#$/]+))'
+
+    _TESTS = [{
+        'url': 'https://rumble.com/c/Styxhexenhammer666',
+        'playlist_mincount': 1160,
+        'info_dict': {
+            'id': 'Styxhexenhammer666',
+        },
+        'expected_warnings': ['Unable to download webpage: HTTP Error 404'],
+    }, {
+        'url': 'https://rumble.com/user/goldenpoodleharleyeuna',
+        'playlist_mincount': 4,
+        'info_dict': {
+            'id': 'goldenpoodleharleyeuna',
+        },
+        'expected_warnings': ['Unable to download webpage: HTTP Error 404'],
+    }]
+
+    def entries(self, url, id):
+        for page in itertools.count(1):
+            webpage = self._download_webpage(f'{url}?page={page}', id, fatal=False, note='Downloading page %d' % page)
+            if not webpage:
+                break
```

Comment:
instead of ignoring all errors, catch only 404

Reply:
That will over complicate things and not needed imo.